lib/ansible/modules/network/exoscale/exo_dns_record.py

```diff
@@ -290,6 +286,13 @@ def __init__(self, module):
         if self.multiple and self.record_type != 'A':
             self.module.fail_json(msg="Multiple is only usable with record_type A")
 
+        self.content = self.module.params.get('content')
+        if self.content and self.record_type == 'TXT':
+            self.content = self.content
```

Comment:
Assigning the same value again, does not make much sense.

~~~python
        self.content = self.module.params.get('content')
        if self.content and self.record_type != 'TXT':
            self.content = self.content.lower()
~~~